octopusdeploy/schema_action_apply_terraform_template.go

```diff
@@ -524,6 +524,14 @@ func flattenApplyTerraformTemplateAction(action *deployments.DeploymentAction) m
 		}
 	}
 
+	if len(action.WorkerPool) > 0 {
```

Comment:
Is there value in this function calling "flattenDeploymentAction", rather than flattenAction on line 497 - or is this a more correct fix?